### DISCUSSION TITLE
Why is Burmilla iso bigger than Boot2Docker? Is there any specific thing that make Burmilla better than Boot2Docker?

### DISCUSSION THREAD

**Discussion by benyaminl** [burmilla/os#139] [General]
hello, I'm quite thankful there are community effort to move Boo2Docker and RancherOS ito new direction, *just I'm late to jump the train.

My Simple question is, what make Burmilla better than B2G? Or it's just for continuing all B2G and follow the Docker update? Thank you

**Comment by olljanat** [comment]
BurmillaOS has nothing to do with Boot2Docker. BurmillaOS continues from where RancherOS left.

I have not actually ever used Boot2Docker but afaiu it focused to provide simple, minimal Linux to run Docker in development environments and which why it was officially replaced by Docker Desktop.

BurmillaOS main purpose instead of (especially on those 1.9.x versions) is to be minimal production grade OS to run Docker and require as little maintaining as possible.

**Reply by benyaminl** [reply]
So it's like Fedora CoreOS? I seen your reply to Boot2Docker last merge on docker toolbox, so I think it's aim on it, seems it's not, but it's still small enough. Thank you for the response.

**Reply by olljanat** [reply]
Similar like Fedora CoreOS yes. However afaik FCOS is quite a lot of bigger than BurmillaOS and because they focus to RedHat stack they prefer Podman instead of Docker (but both was supported last time I checked).

That my comment was related to that discussion that Boot2Docker maintainers proposed migrating to RancherOS and Rancher guys replied that it is also deprecated.

Which one is best for you depends on your use case(s).

**Reply by benyaminl** [reply]
Thank you for the answer @olljanat
